junebug/stores.py

```diff
@@ -19,32 +20,44 @@ def __init__(self, redis, ttl=None):
 
     @inlineCallbacks
     def _redis_op(self, func, id, *args, **kwargs):
+        ttl = kwargs.pop('ttl')
+        if ttl is None:
+            ttl = self.ttl
```

Comment:
Yes this is intentional. `None` means to use the default (and is the default for all of the methods) that is set when you create the class. I'm not sure if it's possible to make this clearer, maybe if the argument doesn't exist, it should use the default, and if it does exist, it uses that value? It would make things a bit more complicated though. Thoughts?